src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -58,11 +60,22 @@ class OutcomeCardContainer extends Component {
     }
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    const curEditingClients = this.state.activeEditingClients;
+    const prevEditingClients = prevState.activeEditingClients;
+    const shouldAnnounceEditing =
+      (curEditingClients.isEmpty() && !prevEditingClients.isEmpty()) ||
```

Comment:
`curEditingClients.isEmpty() !== !prevEditingClients.isEmpty()` might be a little cleaner